app/models/concerns/media_perma_cc_archiver.rb

```diff
@@ -30,7 +30,17 @@ def send_to_perma_cc(url, key_id, _supported = nil)
           data = { location: 'http://perma.cc/' + body['guid'] }
           Media.notify_webhook_and_update_cache('perma_cc', url, data, key_id)
         else
-          raise Pender::Exception::RetryLater, "(#{response.code}) #{response.message}"
+          data = { error: { message: response.message, code: Lapis::ErrorCodes::const_get('ARCHIVER_ERROR') }}
+          Media.notify_webhook_and_update_cache('perma_cc', url, data, key_id)
+          if response&.body.include?("You've reached your usage limit")
```

Comment:
It would be good to have an integration test for this case, just for us to capture if they change their message. On the other hand, this is hard because we would need to have a dedicated account to use only for this test. Hopefully we can track that in the generic `Pender::Exception::PermaCcError` exception report. So let's not change anything for now, this is good to go.